harness/tests/determined/common/experimental/test_trial.py

```diff
@@ -0,0 +1,40 @@
+from typing import Callable
+
+import pytest
+import responses
+
+from determined.common import api
+from determined.common.experimental import trial
+from tests.fixtures import api_responses
+
+_MASTER = "http://localhost:8080"
+
+
+@pytest.fixture
+def standard_session() -> api.Session:
+    return api.Session(master=_MASTER, user=None, auth=None, cert=None)
+
+
+@pytest.fixture
+def make_trialref(standard_session: api.Session) -> Callable[[int], trial.Trial]:
+    def _make_trialref(trial_id: int) -> trial.Trial:
+        return trial.Trial(trial_id=trial_id, session=standard_session)
+
+    return _make_trialref
+
+
+@responses.activate
+def test_trial_includes_summary_metrics(make_trialref: Callable[[int], trial.Trial]) -> None:
+    trial_ref = make_trialref(1)
+
+    tr_resp = api_responses.sample_get_trial()
+
+    trial_url = f"{_MASTER}/api/v1/trials/{trial_ref.id}"
+    responses.get(trial_url, json=tr_resp.to_json())
+
+    trial_ref.reload()
+
+    responses.assert_call_count(trial_url, 1)
```

Comment:
It's hard for me to tell a good story about what's under test, here. I have:
* `reload` calls a specific method in master exactly once
* After reload, `summary` is not None
* After reload, `summary['avg_metrics']` is not None
* After reload, `summary['validation_metrics']` is not None

I'm not sure these specifically test something I want to see tested in `reload`. So... what _would_ I want to see tested in reload? Or alternately, what would I want to see tested wrt summaryMetrics? I'm not really sure. Chat?

Reply:
moved this test into test_determined as we discussed. and also removed asserts on avg_metrics and validation_metrics. the rationale is that this test is simply responsible for making sure the response includes a summary metrics field, not validating what summary metrics should be.

Reply:
I'd still like to see the following:
1. Any tests that won't pass for an arbitrary sample trial have the contents explicitly set, rather than us depending on the sample
2. The test's assertions are no more and no less than what's needed for the test as described in its name